vehicle-python/src/vehicle_lang/compile/abc/translation.py

```diff
@@ -64,36 +64,30 @@ def translate_DefPostulate(
     def translate_expression(
         self, expression: vcl_ast.Expression
     ) -> vcl_var.Expression:
-        if isinstance(expression, vcl_ast.App):
-            return self.translate_App(expression)
         if isinstance(expression, vcl_ast.Var):
             return self.translate_Var(expression)
-        if isinstance(expression, vcl_ast.Builtin):
-            return self.translate_Builtin(expression)
         if isinstance(expression, vcl_ast.Lam):
             return self.translate_Lam(expression)
-        if isinstance(expression, vcl_ast.PartialApp):
-            return self.translate_PartialApp(expression)
         if isinstance(expression, vcl_ast.Pi):
             return self.translate_Pi(expression)
```

Comment:
Unfortunately here we'll have to add all the extra subclasses of `Expression` we've added above.